sumo_mgmt.py

```diff
@@ -743,3 +756,7 @@ def delete_offline_collectors():
             print_sources_table(collectors_with_sources, source_table_headings)
         elif args.deleteOfflineCollectors and prompt(msg):
             delete_offline_collectors()
+        elif args.budgetId:
+            for collector in collectors:
+                print 'assigning budgetId '+args.budgetId[0]+' to collector '+str(collector['name'])
```

Comment:
nit: spaces around `+` like
```
print("Assigning budgetId " + args.budgetId[0] + " to collector " + str(collector['name']))
```

Reply:
we also need to change `print` to `print()` to avoid breaking python3 support

Reply:
Done